hooks/project-relman/code-review-testing.yml

```diff
@@ -39,21 +39,24 @@ payload:
       - $if: firedBy == 'pulseMessage'
         else: {}
         then:
-          $if: {$eval: '"runId" in payload && "status" in payload'}
```

Comment:
Wouldn't it be simpler to just remove the `eval`?

Reply:
You mean this would work ?

```yaml
$if: "runId" in payload && "status" in payload
then:
  ...
```

Reply:
you'd keep the quotes: `$if: '"runId" in payload && "status" in payload'`

Reply:
Ok, I updated the PR to do that. Thanks for your help :+1:

Reply:
Done !